src/pagination/pagination.js

```diff
@@ -32,8 +32,11 @@ angular.module('ui.bootstrap.pagination', [])
     $scope.page = parseInt(ngModelCtrl.$viewValue, 10) || 1;
   };
 
-  $scope.selectPage = function(page) {
+  $scope.selectPage = function(page, evt) {
     if ( $scope.page !== page && page > 0 && page <= $scope.totalPages) {
+      if (evt && evt.target) {
```

Comment:
Yes, below is the only spot where it gets called without an event.

https://github.com/benthepoet/bootstrap/blob/pagination-fix/src/pagination/pagination.js#L59-L68